subprojects/testfx-core/testfx-core.gradle

```diff
@@ -28,6 +28,7 @@ dependencies {
     compile "com.google.guava:guava:21.0"
     compile "org.hamcrest:hamcrest-core:1.3"
     compile 'com.google.code.findbugs:annotations:3.0.1u2'
+    compile 'org.osgi:org.osgi.core:6.0.0'
```

Comment:
I am not sure, but I wonder if the scope of this dependency should be changed to [compileOnly](https://blog.gradle.org/introducing-compile-only-dependencies). Can you see if in doing so it still works for you? It may not be an appropriate usage but I am trying to make the dependency "more optional".

Reply:
Sure thing. Will update this PR.